session.go

```diff
@@ -1520,6 +1555,16 @@ func (b *Batch) GetRoutingKey() ([]byte, error) {
 	return nil, nil
 }
 
+func (b *Batch) isBatchIdempotent() bool {
```

Comment:
Shouldn't a batch be idempotent or not as a whole?

Reply:
yes, of course.  That's exactly what this call is checking. But at the moment there's no use for it, so will remove with other Batch api changes.